bcipy/tasks/session_data.py

```diff
@@ -94,13 +135,16 @@ def stim_evidence(self, alphabet: List[str], n_most_likely: int = 5) -> Dict:
             n_most_likely - number of most likely elements to include
         """
         likelihood = dict(zip(alphabet, self.likelihood))
-        return {
+        data = {
             'stimuli': self.stimuli,
-            'lm_evidence': dict(zip(alphabet, self.lm_evidence)),
-            'eeg_evidence': dict(zip(alphabet, self.eeg_evidence)),
-            'likelihood': likelihood,
-            'most_likely': dict(Counter(likelihood).most_common(n_most_likely))
         }
+        for name, evidence in self.evidences.items():
+            data[Inquiry.serialized_name(name)] = dict(zip(alphabet, evidence))
```

Comment:
I don't understand the reason for the indirection here - why not use `data[name]`?

Reply:
This is mainly for backwards compatibility. In the session data we have called it `eeg_evidence` and in the Copy Phrase Wrapper it's referred to as `ERP`.